web_tree_resize_column/README.rst

```diff
@@ -0,0 +1,91 @@
+==============
+Resize Columns
+==============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_tree_resize_column
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_tree_resize_column
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds the ability to resize columns in tree views.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to any tree view;
+#. hover over the border of the column you want to resize;
+#. hold click and drag the column to the desired width.
+
+This also works for tree views that are nested in form views.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/web/issues/new?body=module:%20web_tree_resize_column%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Onestein
+
+Contributors
+~~~~~~~~~~~~
+
+* Dennis Sluijk <d.sluijk@onestein.nl>
+* Ruchir Shukla <ruchir@bizzappdev.com>
```

Comment:
2 people worked on changed 11.0 to 12.0 in the manifest?

Reply:
review and testing and other things. it was overall just one change. or something. but testing was also done. if you want I can both form the contributors.

Reply:
@tarteo In Contributors, we can add more than one also no need to added one only. If a company is the same then adding two also. 

It's not a big deal

Reply:
@nikul-serpentcs only if actually 2 people worked on it...

Reply:
@tarteo  Yes, It's representing @bizzappdev Company name, So One or more people work on together.

Reply:
removed.

Reply:
It's not a problem if 2 people actually worked on it. But if somebody who tested the module is also considered a contributor than please add these people:

* Yung-Wa Ng <yw.ng@onestein.nl>
* Patrick Mallee <p.mallee@onestein.nl>